docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
-Provide a `description <http://www.meetup.com/Write-The-Docs-PDX/events/231735823/>`_ of your event's topic, a `schedule <http://www.meetup.com/Write-the-Docs-SF/events/232289251/>`_, precise info on where to meet and, if relevant, short `speakers' bio <http://www.meetup.com/Write-The-Docs-London/events/231780773/>`_.
 
 Venue:
 ~~~~~~
 
-Try to find a place that's relatively central or well connected to public transportation.
+Ideal location: an office with easy access to your community via car or
+public transportation.
+
+If your location includes security, tell your members what they need to do to
+access the facility.
 
-Depending on your meetup, here's a list of what you may need:
+Make sure your location includes:
 
-* Electricity and enough power outlets for everyone.
-* A good Internet connection: make the network name and password available. If possible, write it somewhere visible so you don't have to repeat it all the time.
-* A projector: have adapters available for speakers or ask them to bring one.
-* Food and/or beverages: include non-alcoholic and vegetarian options.
+* Wireless Internet Access: announce the network name and password at the start of the meetup.
+* Display equipment such as HDMI access to a monitor, or a projector.
+* Food and/or beverages: if possible, include vegan/vegetarian options, as well as non-alcoholic beverages.
+* Access for anyone who is disabled.
 
-Try to have a venue accessible to people with disabilities: point out in your event description if it's the case or not.
 
-Day of your Meetup Checklist:
------------------------------
+Meetup Day Checklist:
+---------------------
 
-* If you have a Twitter account, tweet about your meetup again and get people excited.
-* Arrive early to your meetup venue.
-* Ask your host to add direction signs with your meetup name in appropriate locations.
+* Arrive early.
+* Provide directions to your Meetup if/as needed.
 * Greet everyone and make them feel welcome.
 * Help your speakers get set up. Put them at ease.
-* If you need help, ask around.
-* Check your event page comments regularly: people having problems can try to contact you there.
+* Monitor the comments on your Meetup page.
+* Live tweet your µeetup.
 
-When your Meetup is Over
-------------------------
 
-* Send a thank you email to your speakers and ask them to post their slides in your event's comments.
-* Send a thank you email to the company who hosted you.
-* If you took pictures, post them on your meetup page. If you plan to do that, always ask people if they're okay with being on a picture that will be published online.
-* If you have a Twitter account: do a tweet to thank everyone who came, your speakers and your sponsor. If your speakers posted their slides online, retweet them.
-* Do a small report to share on meetup-organizers Slack channel.
+Troubleshooting
+---------------
+
+Meetups don't always happen in the way you planned. The following list include
+problems that you may encounter:
+
```

Comment:
Meetups don't always go according to plan. The following list summarizes some of the problems that you might encounter with suggestions for addressing them: